app/registries/foreman/plugin.rb

```diff
@@ -317,6 +317,25 @@ def tests_to_skip(hash)
       end
     end
 
+    # Adds setting definition
+    #
+    # ===== Example
+    #
+    #   settings do
+    #     category(:cfgmgmt, N_('Configuration Management')) do
+    #       setting(:use_cooler_puppet,
+    #         default: true,
+    #         description: N_('This will ensure we install a new and shiny Puppet on hosts instead of a rusty one'),
```

Comment:
Perhaps take what @lzap said and have short lines (which I also like)
```suggestion
    #         description: N_('Use Puppet that goes to 11'),
```

(https://www.youtube.com/watch?v=uMSV4OteqBE)

Reply:
hehe :) right, it will give us bit nicer comment there :+1: